### ISSUE TITLE
Title of audio track does not change when skipping forward or back

### ISSUE THREAD

**Issue by natashache** [tanguyantoine/react-native-music-control#186]
# Description
When using the music control to control a playlist, tapping on the 'forward' button starts the next track in the list. But the track title being displayed still remains the previous track. @tanguyantoine Any idea why it is the case?
![forward](https://user-images.githubusercontent.com/19754530/45688194-be3ecd80-bb1e-11e8-892b-ee2d75de618f.png)

# Platform ?
   - [x ] iOS
   - [ ] Android
   
# Device
  - [ x] Real device

**Comment by RageOfJustice** [comment]
@tanguyantoine Hi! I work on this project. I use [react-native-video](https://github.com/react-native-community/react-native-video) as player
Here is the function that sets MusicControl:

```javascript
export const setBackgroundPlayer = function*(track) {
  const config = {
    title: track.title,
    artist: track.artist,
    album: track.albumTitle,
    artwork: track.trackImageUrl,
    genre: '',
    duration: track.duration,
    description: '',
    color: 0xffffff,
  }

  const trackPosition = yield select(getCurrentTrackPosition)
  const elapsedTime = trackPosition ? trackPosition.playPosition : 0
  const speedRate = yield select(getPlayerSpeedRate)
  MusicControl.setNowPlaying(config)
  MusicControl.updatePlayback({
    state: MusicControl.STATE_PLAYING,
    speed: speedRate,
    elapsedTime,
  })
}
```

I call it from my saga on corresponding action:

```javascript
yield call(setBackgroundPlayer, track)
```

Here is method enabling the controls (I call it in componentDidMount):

```javascript
_setupMusicControls = () => {
    MusicControl.enableBackgroundMode(true)
    // Basic Controls
    MusicControl.enableControl('play', true)
    MusicControl.enableControl('pause', true)
    MusicControl.enableControl('nextTrack', true)
    MusicControl.enableControl('previousTrack', true)

    MusicControl.enableControl('closeNotification', true, { when: 'paused' })

    MusicControl.on('pause', () => {
      this.props.requestPauseTrack(true)
    })
    MusicControl.on('play', () => {
      this.props.requestContinueTrack()
    })
    MusicControl.on('nextTrack', () => {
      this.props.changeTrack(true)
    })
    MusicControl.on('previousTrack', () => {
      this.props.changeTrack(false)
    })
  }
```

Ive tried to call *MusicControl.resetNowPlaying()* before *MusicControl.setNowPlaying(...)* - nothing changes.
Also on Android the same code works fine

**Comment by ReDetection** [comment]
Hi there. I had same issue and I was able to resolve it by always using the `setNowPlaying` and supply all the parameters un there (even those intended to update through `updatePlayback`. Since that all worked like a charm.

Reason why I tried this lays in the fact that [both `setNowPlaying` and `updatePlayback` replace `MPNowPlayingInfoCenter.nowPlayingInfo`](https://github.com/tanguyantoine/react-native-music-control/blob/5f7174c361f31d31f2d4a9dab756bb6cf807edf6/ios/MusicControlManager.m#L58-L100), so if you do `setNowPlaying` once in the beginning and shortly after that do the `updatePlayback`, you will effectively lose what have been set previously – I guess iOS does some throttling.
